cumulus/zombienet/zombienet-sdk/tests/sync_blocks/sync_blocks_from_tip_without_connected_collator.rs

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::anyhow;
+
+use cumulus_zombienet_sdk_helpers::assert_para_throughput;
+use polkadot_primitives::Id as ParaId;
+use subxt::{OnlineClient, PolkadotConfig};
+use zombienet_sdk::{LocalFileSystem, Network, NetworkConfigBuilder};
+
+const PARA_ID: u32 = 2000;
+const BEST_BLOCK_METRIC: &str = "block_height{status=\"best\"}";
+
+#[tokio::test(flavor = "multi_thread")]
+async fn sync_blocks_from_tip_without_connected_collator() -> Result<(), anyhow::Error> {
+	let _ = env_logger::try_init_from_env(
+		env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "info"),
+	);
+
+	log::info!("Spawning network");
+	let network = initialize_network().await?;
+
+	let relay_alice = network.get_node("alice")?;
+
+	let relay_client: OnlineClient<PolkadotConfig> = relay_alice.wait_client().await?;
+
+	log::info!("Ensuring parachain is registered");
+	assert_para_throughput(&relay_client, 2, [(ParaId::from(PARA_ID), 2..3)].into_iter().collect())
```

Comment:
IMO we don't need this one. The one below will already make sure of the registration. Each time we call this, we wait for the next session, which is pretty unnecessary here.

Reply:
Yes, that makes sense. Will rework it